src/checkout/views/Payment/Gateways/Braintree/CreditCard.tsx

```diff
@@ -95,39 +131,64 @@ class CreditCard extends React.PureComponent<ProviderProps, ErrorData> {
 
     return (
       <Form formRef={formRef} onSubmit={this.handleSubmit}>
-        <span className="input__label">Number</span>
-        <div className={classNameError(cardNumber)}>
+        <div
```

Comment:
Maybe we could refactor those 3 divs in form to one functional component with props?:)